<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30)</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>870</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Â© Created by Krylov</string>
   </property>
  </widget>
  <widget class="QLabel" name="standard_welcome">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>LOGGED IN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_image_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>user.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="logged_in_username_editform">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 255, 127);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>DB: GuideCompanies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gobackButton_editform">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>820</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 87 18pt &quot;Arial Black&quot;;
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_editform">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>691</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="current_table">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Table:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>180</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>820</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 229, 255);
font: 87 18pt &quot;Arial Black&quot;;
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>300</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>360</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit5">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>420</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>480</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_foreign_points1">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>550</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 20px;
	color: white;
	padding-left:20px;
	padding-right:20px;
	background-color: rgb(30, 30, 30);
	font-size: 18px;
}
QComboBox QAbstractItemView {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_one">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_two">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_three">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_four">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_five">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_six">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_seven">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>550</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="box_foreign_points2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>590</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 20px;
	color: white;
	padding-left:20px;
	padding-right:20px;
	background-color: rgb(30, 30, 30);
	font-size: 18px;
}
QComboBox QAbstractItemView {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_foreign_points3">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>630</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 20px;
	color: white;
	padding-left:20px;
	padding-right:20px;
	background-color: rgb(30, 30, 30);
	font-size: 18px;
}
QComboBox QAbstractItemView {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_eight">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_nine">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>630</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>680</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45,45,45);
	font-size: 18px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_ten">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>710</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget QToolButton {
	color: white;
}
QCalendarWidget QAbstractItemView:enabled {
	color: black;
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(85, 255, 255);
	border-radius: 2px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
